brotab_ulauncher/extension.py

```diff
@@ -64,17 +62,29 @@ def search_tabs(self, event):
         items = []
         tabs = self.brotab_client.search_tabs(event.get_argument())
 
-        for tab in tabs[:DISPLAY_MAX_RESULTS]:
+        max_results = int(extension.preferences["max_results"])
```

Comment:
No need to pass the extension as an argument. You can use `self.preferences`  instead.

Reply:
Haha, of course, sorry, I didn't make the connection there :sweat_smile:  Fixed now, thanks!

Reply:
Haha, of course, sorry, I didn't make the connection there :sweat_smile: Fixed now, thanks!